rinad/src/ipcm/addons/ma/flowm.cc

```diff
@@ -357,7 +357,7 @@ void* ActiveWorker::run(void* param)
 			LOG_DBG("Connection stablished between MAD and Manager (port id: %u)", port_id);
 
 			//I/O loop
-			while(true) {
+			while(keep_running) {
```

Comment:
This change was not the cause of the problem but I thought it was useful (the outer loop has this condition) but I didn't remember the exception... you are true. I am repushing